frontend/src/sass/components/_buttons.scss

```diff
@@ -7,83 +7,87 @@
    Front Page
    ================================= */
 
-$hero-btn-width: rem-calc(220);
-$hero-btn-height: rem-calc(65);
-
-.btn {
- 
-    width: $hero-btn-width;
-    height: $hero-btn-height; 
-    font-size: rem-calc(24);
-    font-weight: lighter;
+//$hero-btn-width: rem-calc(220);
+//$hero-btn-height: rem-calc(65);
```

Comment:
everything is commented out for now until i finish the tidy up just so i can re-use what i can